lib/hanami/assets/helpers.rb

```diff
@@ -166,18 +166,22 @@ def self.included(base)
       #   <%= javascript 'application' %>
       #
       #   # <script src="https://assets.bookshelf.org/assets/application-28a6b886de2372ee3922fcaf3f78f2d8.js" type="text/javascript"></script>
-      def javascript(*sources, **options)
-        _safe_tags(*sources) do |source|
-          tag_options = options.dup
```

Comment:
@malin-as @AlfonsoUceda This breaks the contract of not mutating the given `options`. Can you please provide an alternative implementation? Thanks.

Reply:
@jodosha thanks for the comment, @malin-as can you create a PR using a non mutable implementation? If you can't, I can do it.

Reply:
Hi,

I made a new PR with non-mutating `options`: https://github.com/hanami/assets/pull/87